src/main/java/net/datafaker/providers/base/Options.java

```diff
@@ -104,6 +104,16 @@ public String option(String... options) {
         return options[faker.random().nextInt(options.length)];
     }
 
+    /**
+     * Returns a random String element from an varargs.
+     *
+     * @param options The varargs to take a random element from.
+     * @return A randomly selected element from the varargs.
+     */
+    public <T> T option(List<T> options) {
+        return options.get(faker.random().nextInt(options.size()));
+    }
```

Comment:
At the very end of this class there is already existing method 
```java
    /**
     * Returns a random element from a list.
     *
     * @param list The list to take a random element from.
     * @param <E>  The type of the elements in the list.
     * @return A randomly selected element from the list.
     */
    public <E> E nextElement(List<E> list) {
        return list.get(faker.random().nextInt(list.size()));
    }
```
will not it help?

We need a separate for `String` because it could be used in expressions, however it's impossible to use generics in expressions... at least right now